core/workloads/containerized_workload/containerized_workload.md

```diff
@@ -0,0 +1,288 @@
+# Containerized Workload
+A containerized workload is used for long-running workloads in containers. Here is the definition of the containerized workload.
+
+```yaml
+apiVersion: core.oam.dev/v1alpha2
+kind: WorkloadDefinition
+metadata:
+  name: containerizedworkload.core.oam.dev
+spec:
+  definitionRef:
+    schemaName: containerizedworkload.core.oam.dev
```

Comment:
```suggestion
    name: containerizedworkload.core.oam.dev
```
per: https://github.com/crossplane/crossplane/pull/1276#discussion_r383647996

Reply:
See this was already pointed out while I was reviewing, resolving in favor of https://github.com/oam-dev/spec/pull/316#discussion_r384720593